### PR TITLE
AuthZ IdP access token sample

### PR DESCRIPTION
**Pull request by ra2085**

### CODE REVIEW

File: authorize-idp-access-tokens/README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Authorize IdP Access Token
 
-This sample allows you to authorize JWT access tokens issued by an OIDC compliant identity provider.
+This sample allows you to authorize JWT access tokens issued by an OpenID Identity Provider.
+
+## About OIDC
+
+[OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html) is an identity layer on top of the OAuth 2.0. It allows Relying Parities (clients) to verify the identity of an End-User based on the authentication performed by an Authorization Server, as well as to obtain profile information about the End-User.
```
Comment (ssvaidyanathan) [inline authorize-idp-access-tokens/README.md:7]: "parties"

File: authorize-idp-access-tokens/README.md [hunk 2]
```diff
@@ -15,16 +19,17 @@ JWT tokens are used by [OIDC](https://openid.net/specs/openid-connect-core-1_0.h
 
 With this example you'll be able to:
 
-* Import Client IDs and Secrets from an Identity Provider into Apigee as Developer App credentials
-* Validate JWT access tokens issued by an Identity Provider that supports [OIDC](https://openid.net/specs/openid-connect-core-1_0.html) or [RFC9068](https://datatracker.ietf.org/doc/rfc9068/)
-* Identify the Client App in Apigee
+* Import Client IDs and Secret representing a Relying Party or Client Application from an OpenID Identity Provider that supports JWT access tokens based on [RFC9068](https://datatracker.ietf.org/doc/rfc9068/)
```
Comment (ssvaidyanathan) [inline authorize-idp-access-tokens/README.md:22]: Client ID (remove the "s")

Comment (ssvaidyanathan) [reply authorize-idp-access-tokens/README.md:22]: Should it OpenID Provider. OpenID Identity Provider sounds redundant

File: authorize-idp-access-tokens/README.md [hunk 3]
```diff
@@ -36,9 +41,7 @@ This implementation includes two artifacts:
     * curl
     * jq
     * npm
-5. Configure and obtain Client App credentials from the Identity Provider. These usually are client identifier and secret / password credential pairs.
-
-6. Obtain a JWT access token from the Identity Provider. The grant type and method to obtain the access token is specific to each IdP. However, it is common for Identity Providers to support the client_credentials grant type to issue access tokens to authorized applications.
+5. Obtain a Client ID and a JWT access token from an OpenID Identity Provider (you'll be using the values during the quickstart)
```
Comment (ssvaidyanathan) [inline authorize-idp-access-tokens/README.md:44]: Should it OpenID Provider. OpenID Identity Provider sounds redundant

File: authorize-idp-access-tokens/README.md [hunk 4]
```diff
@@ -61,7 +64,7 @@ cd apigee-samples/authorize-idp-access-tokens
 * `PROJECT` the project where your Apigee organization is located
 * `APIGEE_ENV` the Apigee environment where the demo resources should be created
 * `JWKS_URI` the [JWKS URI](https://openid.net/specs/openid-connect-core-1_0.html#RotateSigKeys) from the OIDC identity provider that will issue JWT access tokens
-* `TOKEN_ISSUER` the [Issuer Identifier](https://openid.net/specs/openid-connect-core-1_0.html#IssuerIdentifier) from the OIDC Identity provider that will issue JWT access tokens
+* `TOKEN_ISSUER` the [Issuer Identifier](https://openid.net/specs/openid-connect-core-1_0.html#IssuerIdentifier) from the OpenID Identity Provider that will issue JWT access tokens
```
Comment (ssvaidyanathan) [inline authorize-idp-access-tokens/README.md:67]: Should it OpenID Provider. OpenID Identity Provider sounds redundant

File: authorize-idp-access-tokens/README.md (hunk 3 above)
Comment (ra2085) [reply authorize-idp-access-tokens/README.md:44]: I've found multiple public reference using OpenID Connect Identity Provider and OpenID Identity Provider. I could use the former for completion sake.

Comment (ssvaidyanathan) [reply authorize-idp-access-tokens/README.md:44]: lets use the former

File: authorize-idp-access-tokens/clean-up-authorize-idp-access-tokens.sh [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+rm idp_configuration.properties
+
+echo "Undeploying authorization-server-mock proxy"
+REV=$(apigeecli envs deployments get --env "$APIGEE_ENV" --org "$PROJECT" --token "$TOKEN" --disable-check | jq .'deployments[]| select(.apiProxy=="authorization-server-mock").revision' -r)
+apigeecli apis undeploy --name authorization-server-mock --env "$APIGEE_ENV" --rev "$REV" --org "$PROJECT" --token "$TOKEN"
+
+echo "Deleting proxy authorization-server-mock proxy"
+apigeecli apis delete --name authorization-server-mock --org "$PROJECT" --token "$TOKEN"
+
+echo "Deleting mock config environemnt property set..."
```
Comment (drush1980) [inline authorize-idp-access-tokens/clean-up-authorize-idp-access-tokens.sh:70]: `environment`

**Review by drush1980 — APPROVED** [review]
lgtm

**Pull request closed without merging**